data/structure.json

```diff
@@ -1,72 +1,64 @@
 {
-  "menu": [
```

Comment:
Please revert last pull request because it was not reviewed https://github.com/VSlobolinskyi/renome_vue_clean/pull/2